### PR TITLE
[BZ1256688]  CDI and Secure embedded quickstarts unable to deploy

### PR DESCRIPTION
**Pull request by Holmistr**
See: https://bugzilla.redhat.com/show_bug.cgi?id=1256688

### CODE REVIEW

File: cdi/pom.xml [hunk 1]
```diff
@@ -88,7 +88,6 @@
         <dependency>
             <groupId>javax.enterprise</groupId>
             <artifactId>cdi-api</artifactId>
-            <scope>provided</scope>
         </dependency>
```
Comment (dmstahl) [inline cdi/pom.xml]: Should a dependency on javax.cache:cache-api also be added here?

Comment (Holmistr) [inline cdi/pom.xml]: Hmm, I definitely wanted to add also this dependency, I accidentally missed it while committing. However, when I try to build the current quickstarts with latest release (6.6.0.DR1), the quickstart works fine. @slaskawi Could you please confirm that the change we were discussion (about missing dependency) will be also present in next release of 6.6?

Comment (dmstahl) [inline cdi/pom.xml]: The quickstart doesn't build against the latest infinispan SNAPSHOT on the jdg-6.6.x branch (which will be part of 6.6.0.DR2) without this dependency.

Comment (Holmistr) [inline cdi/pom.xml]: Ok, so that's confirmation. Sorry about this, my mistake. I will repush it in a minute.

**Comment by Holmistr** [comment]
@dmstahl re-pushed, it should be ok now :)

**Comment by dmstahl** [comment]
Integrated.  Thanks @Holmistr !

**Pull request merged**